.gitignore.dist

```diff
@@ -0,0 +1,13 @@
+.DS_Store
+.vscode
+Thumbs.db
+phpcs.xml
+phpunit.xml
+wp-cli.local.yml
+node_modules/
+*.sql
+*.tar.gz
+*.zip
+
+# Client build files
```

Comment:
I might recommend removing or renaming `Client build files`, as it is no longer excluding 'Client build files' .. in fact, the comment is quite contrary to what it is doing...  /vendor could, for all intents, be listed right alongside node_modules/ or the others if needed...